library/returnBook/ReturnBookUI.java

```diff
@@ -4,17 +4,17 @@
 
 public class ReturnBookUI {

```

Comment:
Enum names are to start with an uppercase letter and to be in CamelBack
Ex: private enum **ControlState** {INITIALISED, READY, RESTRICTED};

Reply:
Ok Noted. Will make changes